homeassistant/components/livisi/strings.json

```diff
@@ -7,12 +7,19 @@
           "host": "[%key:common::config_flow::data::ip%]",
           "password": "[%key:common::config_flow::data::password%]"
         }
+      },
+      "reauth_confirm": {
+        "title": "[%key:common::config_flow::title::reauth%]",
+        "description": "The LIVISI integration needs to re-authenticate your account"
```

Comment:
Please adjust also the english translation file.

Reply:
I added the changes in the English translation file.